closure/testing/test/BUILD

```diff
@@ -83,6 +83,22 @@ closure_js_test(
     ],
 )
 
+closure_js_library(
+    name = "goog_testSuite_lib",
+    srcs = ["goog_testSuite.js"],
+)
+
+closure_js_test(
+    name = "goog_testSuite_test",
+    srcs = ["goog_testSuite_test.js"],
+    entry_points = {
```

Comment:
To double check, does your test fail when we remove entry_points?

Reply:
Yes, I've verified the failure without the specific `entry_points` entry.